src/core_atmosphere/physics/mpas_atmphys_driver_cloudiness.F

```diff
@@ -121,7 +121,7 @@ subroutine cloudiness_from_MPAS(configs,mesh,diag_physics,sfc_input,its,ite)
 
  call mpas_pool_get_array_gpu(sfc_input,'xland',xland)
  coupler => areaCellField%block%domain%mpas_cpl
- if (coupler%role_is(ROLE_INTEGRATE)) then
+ if (coupler%role_includes(ROLE_INTEGRATE)) then
```

Comment:
Is there really any sense in changing this from `role_is` to `role_includes`? If the role of a task is not *exclusively* integration, then it means the task is also handling radiation and cloud fraction calculations, in which case it doesn't seem like any update of fields should be needed. Did you find a specific need to change the conditional here to use `role_includes`?

Reply:
That was something I recommended. Since there is a chance that we will have the rrtmgf being introduced, we may need to run the radiation on the same roles as GPU. In case of a debug, if we ran the simulation without specifying the roles (ROLE03), then role_is would decline the condition but role_includes would pass the condition to keep cloudiness on the CPU. Am I right? That is why I asked for the switch.

Reply:
I think that makes sense. Thanks for the explanation!